src/main/java/uk/co/caprica/vlcjinfo/MediaInfo.java

```diff
@@ -144,6 +144,10 @@ public void dump(Writer writer) {
         }
     }
 
+    public Map<String, Sections> getSections() {
```

Comment:
I prefer not to use getters per se, so sections() or actually I think sectionsByType() would be preferred.

Reply:
Happy to change this for consistency since it is your project, but [basic OO principles](https://medium.com/swlh/the-four-pillars-of-object-oriented-programming-8ce8a26b32eb) promote the use of verbs for methods ... it's just a widely adopted norm. But hey... your project,  your rules ;-)

Reply:
This is not my first rodeo... In some schools of thought around OO, getters and setters are considered an anti-pattern, but this is not the place for those discussions ;-)